crypto/dsa/dsa_aid.c

```diff
@@ -18,15 +18,15 @@
 /* dsaWithSHA OIDs are of the form: (1 3 14 3 2 |n|) */
 #define ENCODE_ALGORITHMIDENTIFIER_SHA(name, n)                         \
     static const unsigned char algorithmidentifier_##name##_der[] = {   \
-        ASN1_SEQUENCE, 0x06,                                            \
-          ASN1_OID, 0x09, 1 * 40 + 3, 14, 3, n,                         \
+        ASN1_SEQUENCE, 0x07,                                            \
+          ASN1_OID, 0x09, 1 * 40 + 3, 14, 3, 2, n                       \
```

Comment:
Is the 0x09 supposed to be 0x05
```suggestion
          ASN1_OID, 0x09, 1 * 40 + 3, 14, 3, 2, n                       \
```

Reply:
Yes.  Fixed.  Thanks for being eagle-eyed :-)